### PR TITLE
LKE-3682 Updated README.md

### PR DESCRIPTION
**Pull request by daemercado**

### CODE REVIEW

**Comment by daemercado** [comment]
@aminerhanemi Updated the How To and Limitations. Do edit if I forgot something and update the other sections (installation etc)

**Review by aminerhanemi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -98,35 +75,91 @@ Otherwise, go to step 6
...
+      Plausible if your session has been revoked)
+      
+   h. Error unknown
+      Default message if error is not known
 
-9. In case of edges, select the correct identifiers for the source and destination nodes
+## 4. Limitations
 
-10. Click on `Run`
+1. The file size limit is currently set to 80KB.
```
Comment (aminerhanemi) [inline README.md:153]: we can set it back to 3.5Mb

**Review by aminerhanemi — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,132 +1,106 @@
...
+## Set up the Plugin
 
-4. Scroll to the `Plugin settings` field
+> Make sure to have write access to the folder of Linkurious Enterprise as well as a Linkurious admin account.
 
-5. If there are no other plugins configured here, replace the whole content of the field with the following code and skip to step 7. \
-Otherwise, go to step 6
+1. Download the `csv-importer.lke` plugin available [here](https://github.com/Linkurious/lke-plugin-csv-importer/releases/tag/v1.0.0).
+2. Copy `csv-importer.lke` file to the following path `<linkurious>/data/plugins`
+3. Restart Linkurious Enteprise
```
Comment (aminerhanemi) [inline README.md:32]: in the step 6 the user will update the config, so maybe there is no need to restart linkurious here ?

or leave this step and remove the last one as the default config will be added automatically

File: README.md [hunk 3]
```diff
@@ -1,132 +1,106 @@
...
+      "importer": {
         "basePath": "importer"
+      }
     }
     ```
-
 7. Click on `Save`
 
-8. Done!
+## Access the plugin
```
Comment (aminerhanemi) [inline README.md:46]: Having a CA is not a condition to access the plugin. should we just add the link here and as a more advanced option talk about CA?

**Review by aminerhanemi — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (francescoinfante) [reply README.md:46]: @aminerhanemi it kind of his. you need the source key in the URL. alternative is to create the URL yourself with the source key that seemed complicated to explain :D 
you are welcome to propose a sentence though

**Review by aminerhanemi — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,132 +1,108 @@
...
+After the import, you will get one of the following results:
 
-2. Execute the `Import data` custom action
+1. Successful: All nodes / edges have been imported
 
-3. Select the *csv* file from your computer and select the checkbox if this contains an *header*
+2. Failed: Nothing has been imported
 
-4. Click on `Load this file`
+3. Incomplete: Some nodes/edges failed to be imported due to one of the following reasons:
```
Comment (renato-rg) [inline README.md:80]: There is too much text here, making bold `Incomplete:` would help reading, and `b. Unexpected properties (in strict schema)` too

File: README.md [hunk 5]
```diff
@@ -1,132 +1,108 @@
...
 
-Two sample CSV files are available [here](https://github.com/Linkurious/lke-plugin-csv-importer/tree/master/sample%20csv):
-- `person.csv` contains information about person entities (a *node*);
-- `ping.csv` contains information about relationships between two person nodes (a *relationship*).
+The maximum file size currently supported is 3.5MB.
 
-All the information contained in these files has been created with [Mockaroo](http://mockaroo.com/): an online tool for creating random and mock datasets.
\ No newline at end of file
+## Licensing
+The Linkurious CSV importer plugin is licensed under the Apache License, Version 2.0. See LICENSE for the full license text.
```
Comment (renato-rg) [inline README.md:108]: Missing link for `LICENSE`

File: README.md [hunk 6]
```diff
@@ -1,132 +1,108 @@
...
 
-### Prerequisites:
-- a  Linkurious Enterprise 2.9.x server
+![](readme_assets/plugin.png)
 
-## Topics:
-1. [Introduction](#1-introduction)
+### Prerequisites
+- Linkurious Enterprise 2.9.x or above
+- CSV files containing nodes or edges structured as follows:
```
[the remark this answers is not in the document]
Comment (francescoinfante) [reply README.md:12]: oooooh yes, it works only for Neo4j if you write Cypher query directly :D

**Review by renato-rg — APPROVED** [review]

**Pull request merged**